Add: Spawners and landmarks icons

SAADf603 opened the pull request:
## Описание
Добавляет новые и не очень иконки для всех(почти) типов спавнеров и лендмарков, убирает эмо-текстуры все на дельте/кебериаде, большую часть на ЦентКоме, убирает найденные и неиспользуемые дубликаты иконок, включает лендмарки и спавнеры в отображения рендера карт.

## Ссылка на предложение/Причина создания ПР
Сейчас все рендеры карт выглядят интуитивно непонятно для обычных игроков из-за обилия синих, красных и зелёных крестов, которые могут означать что-угодно, что я и взялась исправлять.

Почему более не тестмерж? Лендмарки работают от параметра "name", на локальном сервере было проверено, профессии работают как надо. События так же исправно спавнят кого надо.
Если всё будет работать как надо, то я сделаю второй ПР, в котором закончу с ЦК и гейтами.

## Демонстрация изменений
Новые спавнеры:
![Безымянный](https://user-images.githubusercontent.com/110329212/221656918-dfa0466d-6351-40a5-bc93-328f2e00bb73.png)
Новые лендмарки:
![1](https://user-images.githubusercontent.com/110329212/221657158-a30a2bd4-880f-4c26-a01a-627ab2ad9a85.png)
![2](https://user-images.githubusercontent.com/110329212/221657033-f68eb084-a7f0-46df-84ba-954aeaac71a5.png)
Как будут выглядеть рендеры:
https://discord.com/channels/617003227182792704/1077730897715019776/1079840192292196422

SAADf603 committed:
Relise

SAADf603 committed:
Marauder_Exit fix

Ykipss commented:
А не будет ли крестика на крестике на крестике на крестике на крестике? :XMM:

SAADf603 committed:
missed icon_state fix

SAADf603 commented:
> А не будет ли крестика на крестике на крестике на крестике на крестике? :XMM:

Не должно. У нас не так много спавнеров накладываются друг на друга. А лендмарки вообще по одному расположены.

SAADf603 committed:
satelite old robots fix

SAADf603 commented:
а, ну да, пупс блять........

SAADf603 commented:
*sigh

SAADf603 committed:
conflict fix

SAADf603 committed:
NO MORE PUPS BLYAD(conflict fixes)

SAADf603 committed:
little fix

SAADf603 commented:
В общем и целом готово.

SAADf603 committed:
Merge branch 'ss220-space:master220' into spawners-icons

SAADf603 committed:
damaged and burned turfs return

SAADf603 committed:
Merge branch 'ss220-space:master220' into spawners-icons

Dimach commented on code/game/objects/effects/landmarks.dm:
```diff
@@ -105,25 +106,368 @@
+	name = "Observer-Start"
+	icon_state = "observer_start"
+
+/obj/effect/landmark/JoinLateCyborg
```
Почему название типа с большой буквы? Только нижний регистр!

SAADf603 replied:
@Dimach исправлено и доработано

SAADf603 committed:
register, SCentCom, remains + monkeys

SAADf603 committed:
icon fix

SAADf603 committed:
Merge branch 'ss220-space:master220' into spawners-icons

SAADf603 committed:
tag fix

кто бы знал... что бы не ломать "особенности" спавнов прописала им тэг ручками.

SAADf603 committed:
Merge branch 'spawners-icons' of https://github.com/SAADf603/Paradise into spawners-icons

Dimach commented on code/game/objects/effects/landmarks.dm:
```diff
@@ -105,25 +106,376 @@
-	icon = 'icons/mob/screen_gen.dmi'
-	icon_state = "x"
-	anchored = 1.0
+//Прописано исключительно для включения в рендеры
+//В порядке списка проффессий в лобби
```
```suggestion
//Landmark icons added only for renders
//Same orders as jobs in lobby
```

Dimach committed:
Update code/game/objects/effects/landmarks.dm
